ibis/mapd/tests/test_operations.py

```diff
@@ -134,3 +134,17 @@ def test_literal_geospatial():
         "SELECT 'MULTIPOLYGON(((0 0, 0 0), (0 0, 0 0)), "
         "((0 0, 0 0), (0 0, 0 0)))' AS tmp"
     )
+
+
+@pytest.mark.parametrize(('result_fn', 'expected_fn'), [
+    param(
+        lambda t: t.double_col.arbitrary(),
+        lambda t: t.double_col.iloc[-1],
```

Comment:
Just to confirm, the semantics here are that the last non null value is returned?

Reply:
well I just tried to follow the tests on `tests.all` for `arbitrary` and in the case of omnisci/mapd the default `how` would be `last`.

Reply:
Can you add a test for `'first'` as well, if that exists in mapd?